src/sage/matroids/database/driver_functions.py

```diff
@@ -0,0 +1,165 @@
+r"""
+Functions that access the collections of matroids
+
+This module contains driver functions to easily access the collections of
+matroids in the database. Each of these functions returns a complete list
+of matroids from the corresponding collection. These functions can be viewed
+by typing ``matroids.`` and hitting the ``tab`` button.
+
+AUTHORS:
+- Giorgos Mousa (2023-12-08): initial version
+
+Functions
+=========
```

Comment:
Please remove all of these in all of the files. I think this is useless and is largely inconsistent with the rest of Sage's documentation.